helm/cluster-vsphere/values.schema.json

```diff
@@ -132,6 +158,99 @@
+                                                        "name": "10-coreos-metadata.conf"
+                                                    }
+                                                ],
+                                                "enabled": true,
+                                                "name": "coreos-metadata.service"
+                                            },
+                                            {
+                                                "contents": {
+                                                    "install": {
+                                                        "wantedBy": [
+                                                            "multi-user.target"
+                                                        ]
+                                                    },
+                                                    "unit": {
+                                                        "description": "Set machine hostname"
+                                                    }
+                                                },
+                                                "dropins": [
+                                                    {
+                                                        "contents": "[Unit]\nRequires=coreos-metadata.service\nAfter=coreos-metadata.service\n[Service]\nType=oneshot\nRemainAfterExit=yes\nEnvironmentFile=/run/metadata/coreos\nExecStart=/opt/bin/set-hostname",
+                                                        "name": "10-set-hostname.conf"
+                                                    }
+                                                ],
+                                                "enabled": true,
+                                                "name": "set-hostname.service"
+                                            },
+                                            {
+                                                "contents": {
+                                                    "install": {
+                                                        "wantedBy": [
+                                                            "default.target"
+                                                        ]
+                                                    },
+                                                    "unit": {
+                                                        "description": "Disable TCP segmentation offloading"
+                                                    }
+                                                },
+                                                "dropins": [
+                                                    {
+                                                        "contents": "[Unit]\nAfter=network.target\n[Service]\nType=oneshot\nRemainAfterExit=yes\nExecStart=/usr/sbin/ethtool -K ens192 tx-udp_tnl-csum-segmentation off\nExecStart=/usr/sbin/ethtool -K ens192 tx-udp_tnl-segmentation off",
```

Comment:
we should revisit this later and turn it into a script which gets called by the `Service` - at the moment we're hard-coding the NIC name in which will probably bite us in the future.